text/0000-microsoft-jdk.md

```diff
@@ -0,0 +1,39 @@
+# Paketo Community Microsoft OpenJDK Buildpack
+
+## Summary
+
+A [Microsoft OpenJDK Buildpack](https://github.com/eddumelendez/microsoft-openjdk) for supplying Microsoft's distribution of OpenJDK has been created by [@eddumelendez][@eddumelendez]. It is built using common tooling (libpak and lijvm) used by the Paketo Java sub-team and should be adopted into the Paketo Community Github Org.
+
+## Motivation
+
+We would like to move the Microsoft OpenJDK Buildpack into the Paketo Community so that the community can use it to build Java applications, in particular on Microsoft Azure. This will also give the buildpack maintainer the ability to leverage common Paketo tooling to keep the buildpack dependencies updated and release the buildpack.
```

Comment:
I think this can go directly into `paketo` and skip `paketo-community`. This buildpack is nearly identical other JVM buildpacks and therefore the risk here seems very low and this shouldn't need to incubate in community.

Reply:
Done.